workflow/github.md

```diff
@@ -1,2 +1,122 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+- [Working on GitHub](#working-on-github)
+  * [Table of content](#table-of-content)
+  * [How can I contribute?](#how-can-i-contribute-)
+    + [Report bugs](#report-bugs)
+    + [Suggest enhancements](#suggest-enhancements)
+    + [Work on an issue](#work-on-an-issue)
+  * [Pull requests](#pull-requests)
+    + [First contribution](#first-contribution)
+    + [Styleguides](#styleguides)
+    + [Merging process](#merging-process)
+  * [Templates](#templates)
+    + [Issues template](#issues-template)
+    + [Pull request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
```

Comment:
I'd like this to be even more succinct - it still seems like an unnecessary list of things. I'd go with something like:

> "... and open an issue. Describe your bug or feature in the Title field and follow (the template)(LINK)."

I don't think we need to mention Labels, as this section is clearly geared towards outside contributors and they don't have permission to add labels, and I don't think we need to explicitly tell them to submit it.

But I'm also happy to merge it as it is and I can make a follow-up PR with my ideas here.

Reply:
I don't think it is really shocking, maybe we can keep it merge it and discuss about this later.

Reply:
Okay